packages/amplify_datastore/example/lib/main.dart

```diff
@@ -90,10 +91,10 @@ class _MyAppState extends State<MyApp> {
   // Platform messages are asynchronous, so we initialize in an async method.
   Future<void> initPlatformState() async {
     datastorePlugin = AmplifyDataStore(modelProvider: ModelProvider.instance);
-    listenToHub();
     await Amplify.addPlugin(datastorePlugin);
     // Configure
     await Amplify.configure(amplifyconfig);
+    listenToHub();
```

Comment:
do android/iOS prevent from listening to hub events prior to configure?

Reply:
No, customers can establish a listener as long as the plugin is added.  Once we add the 'configure' events, they will be able to listen for those.